src/app/mock/mock-http.ts

```diff
@@ -247,10 +247,6 @@ export class MockHttp extends HttpService {
           } else {
             return this.createResponse(url.toString(), 200, 'ok', { data: this.mockDataService.getAllAreas() });
           }
-        case '/source-link-types':
```

Comment:
We will need **/workitemlinktypes** mock data service. Maybe just keep one of these and replace **/target-link-types** with **/workitemlinktypes**